README.md

```diff
@@ -4,6 +4,58 @@ A lightweight IPNI node mapping Filecoin PieceCID → payload block CID.
 
 - [Design doc](./docs/design.md)
 
+## Basic use
+
+Note: this will change soon, see
+https://github.com/filecoin-station/piece-indexer/issues/33
+
+## `GET /sample/{provider-id}/{piece-cid}
+
+Sample a set of multihashes ingested by IPNI for a given ContextID.
```

Comment:
Why does this sample a set of multihashes for a given ContextID, and not return all multihashes known by this service?

Maybe you didn't mean to use "to sample" here?

Reply:
I am cloning the future IPNI Reverse Index. I copied this sentence from their OpenAPI spec.

A single ContextID often has many millions of payload CIDs. It's not feasible for the real IPNI to return all payload CIDs, it will always return only a sample.

In the XEDNI spec, there are two more request parameters that we ignore here: `seed` and `max`. The first provides randomness, and the second defines the sample size.

Can you suggest a better description I can put here?

Reply:
Got it! What about this?
```suggestion
Sample a set of multihashes ingested by IPNI for a given ContextID, matching the route eventually to be exposed by an IPNI reverse index.
```

Reply:
Great, thank you!